common/djangoapps/student/migrations/0046_auto__add_entranceexamconfiguration__add_unique_entranceexamconfigurat.py

```diff
@@ -121,9 +121,11 @@ def backwards(self, orm):
             'Meta': {'object_name': 'LinkedInAddToProfileConfiguration'},
             'change_date': ('django.db.models.fields.DateTimeField', [], {'auto_now_add': 'True', 'blank': 'True'}),
             'changed_by': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['auth.User']", 'null': 'True', 'on_delete': 'models.PROTECT'}),
-            'dashboard_tracking_code': ('django.db.models.fields.TextField', [], {'blank': 'True'}),
+            'company_identifier': ('django.db.models.fields.TextField', [], {}),
```

Comment:
It's not-- this is the Cale commit that is necessary for migrations to work. The comment got lost in the PR change.